README.md

```diff
@@ -1 +1,52 @@
-# the-concert-project
\ No newline at end of file
+# the-concert-project
+
```

Comment:
- [ ] OPTIONAL: I suggest to update title to make your readme look professional. You can capitalise the heading and delete dashes 😉